src/rosdep2/main.py

```diff
@@ -682,7 +682,8 @@ def update_error_handler(data_source, exc):
 
 
 def command_keys(lookup, packages, options):
```

Comment:
It's unfortunate that lookup isn't an optional kwarg like the other command_ functions. I'm not sure it's worth an API-break to make them consistent but I could see adding a TODO or Issue to update it if we ever do an API review and major-bump.

Reply:
There are a couple of "classes" of the command functions in this file. My first run at this change was to converge them into a single function signature, but it would have been tricky to maintain API backward compatibility, so I punted it.

In short, I agree. It would be nice to clean this up someday.